Code/V2/src/themes/index.js

```diff
@@ -0,0 +1,27 @@
+/* 
+    script: color of the script on a key (number, letter, etc.)
+    key: color of a key in a neutral state
+    flashed: color of a key while flashed
+    selected: color of a key currently selected by the user
+*/
+
+export const defaultTheme = {
```

Comment:
add the keyword "default" here (it will be our default export)

Reply:
Overall this is looking good! We shouldn't forget to add contants for the textfield's colors (its text, background & border).

Reply:
To help show the separation of the colors for a textfield vs a key, you can even use this structure:

defaultTheme = {
    key: {
        ....
    },
    inputBar: {
        .....
    },
}

Reply:
Maybe also include the view's background colour (I understand a lot of these colours will be exactly the same - cyan and white, but it's nice to keep the separation)